packages/nimble-components/src/table/tests/table-column-sizing.spec.ts

```diff
@@ -869,4 +869,12 @@ describe('Table Interactive Column Sizing', () => {
         expect(actualFractionalWidths).toEqual(expectedFractionalWidths);
         expect(actualPixelWidths).toEqual(expectedPixelWidths);
     });
+
+    it('sizing to left when columns are all on sub-pixel boundary and just above minimum size, does not cause total column width to change', async () => {
+        await pageObject.sizeTableToGivenRowWidth(202, element); // columns now on half-pixel boundary
```

Comment:
Is sizing the table to a 202px row width a guarantee that columns are now on a half-pixel boundary?
Isn't that a function of a bunch of things related to the styling, etc?

How do you feel about coming up with an assertion that can validation the assumption that the table is actually at a half-pixel boundary before the test runs. Alternatively is there a test set-up that would give a clear guarantee the setup is what we expect?

Reply:
It's a guarantee with the current test setup, where there are four columns that should each be sized to 50.5 pixels since they are equally sized. I can provide an assertion to this effect prior to sizing a column.